ext/js/data/anki-util.js

```diff
@@ -19,7 +19,7 @@
 import {isObjectNotArray} from '../core/object-utilities.js';
 
 /** @type {RegExp} @readonly */
-const markerPattern = /\{([\w-]+)\}/g;
+const markerPattern = /\{([\p{Letter}\p{Number}_-]+)\}/gu;
```

Comment:
This is just a QOL change with no functional change? I can't find the documentation for this 😅